src/components/textInput.re

```diff
@@ -1,9 +1,8 @@
 [@bs.module "react-native"]
 external view : ReasonReact.reactClass = "TextInput";
 
-[@bs.send] external _isFocused : ReasonReact.reactRef => Js.boolean = "";
+[@bs.send] external isFocused : ReasonReact.reactRef => bool = "";
 
-let isFocused = UtilsRN.(Js.to_bool << _isFocused);
```

Comment:
Please double check this one, not sure what UtilsRN. is doing

Reply:
I think it is just doing pointfree composition